libraries/botbuilder-adapter-twilio-whatsapp/src/twilioWhatsAppAdapter.ts

```diff
@@ -57,24 +94,51 @@ export class TwilioWhatsAppAdapter extends BotAdapter {
     protected readonly client: Twilio.Twilio;
     protected readonly channel: string = 'whatsapp';
 
+    // OAuth Properties
+    protected readonly credentials: AppCredentials;
```

Comment:
LGTM. Eventually it would be great to have all OAuth stuff moved to the CustomWebAdapter (#141), with an optional flag to enable OAuth.

Reply:
I will take a look into it (#141) this weekend if I can.